src/ansys/dpf/composites/data_sources.py

```diff
@@ -38,13 +39,43 @@ class CompositeDefinitionFiles:
 class ContinuousFiberCompositesFiles:
     """Provides the container for continuous fiber composite file paths."""
 
-    rst: _PATH
+    rst: List[_PATH]
     composite: Dict[str, CompositeDefinitionFiles]
     engineering_data: _PATH
     # True if files are local and false if files
     # have already been uploaded to the server
     files_are_local: bool = True
 
+    def __init__(
+        self,
+        rst: Union[List[_PATH], _PATH],
+        composite: Dict[str, CompositeDefinitionFiles],
+        engineering_data: _PATH,
+        files_are_local: bool = True,
+    ) -> None:
+        """Initialize the ContinuousFiberCompositesFiles container.
+
+        Parameters
+        ----------
+        rst :
+            A single path to an RST file, or a list of paths to distributed
+            RST files.
+        composite :
+            Dictionary of composite definition files. The key can be chosen
+            freely.
+        engineering_data :
+            Path to the engineering data file.
+        files_are_local :
+            True if files are on the local machine, False if they have already
+            been uploaded to the DPF server..
+        """
+        if isinstance(rst, (str, pathlib.Path)):
+            rst = [rst]
+        self.rst = rst  # type: ignore
+        self.composite = composite
+        self.engineering_data = engineering_data
+        self.files_are_local = files_are_local
+
 
 @dataclass
 class ShortFiberCompositesFiles:
```

Comment:
Should this data class be updated as well?

Reply:
Hm, I haven't tested distributed RST in the short fiber context at all yet (neither here, nor in the back-end). Could still do this of course, but then we may as well create a separate PR.

Reply:
OK. We can inform Federico that distributed RST are supported now and that he can test it for his operators.